### PR TITLE
[Localization] ME translations - The sequel

### PR DESCRIPTION
**Pull request by Adri1**
# PR FROZEN, NO MORE TRANSLATION WILL BE ACCEPTED FOR NOW

## What are the changes the user will see?
Translated ME

**Completed languages:**
- [X] German **- Already completed in PR #4183**
- [ ] Spanish _- partial translation
- [X] French **- Already completed in PR #4183**
- [ ] Italian _- superficial translation_
- [x] Japanese
- [X] Korean **- Already completed in PR #4183**
- [X] Brazilian Portugese **- Already completed in PR #4183**
- [X] Chinese (Simplified) **- Already completed in PR #4183**
- [ ] Chinese (Traditional) _- nothing done_

## Why am I making these changes?
To have ME translated

## What are the changes from a developer perspective?
None

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/ja/mystery-encounters/fiery-fallout-dialogue.json [hunk 1]
```diff
@@ -1 +1,26 @@
...
+    },
+    "3": {
+      "label": "Your Fire Types Help",
+      "tooltip": "(+) End the conditions\n(+) Gain a Charcoal",
+      "disabled_tooltip": "You need at least 2 Fire Type Pokémon to choose this",
+      "selected": "Your {{option3PrimaryName}} and {{option3SecondaryName}} guide you to where two {{volcaronaName}}s are in the middle of a mating dance!$Thankfully, your Pokémon are able to calm them,\nand they depart without issue."
+    }
+  },
+  "found_charcoal": "After the weather clears,\nyour {{leadPokemon}} spots something on the ground.$@s{item_fanfare}{{leadPokemon}} gained a Charcoal!"
+}
```
Comment (Chapybara-jp) [inline src/locales/ja/mystery-encounters/fiery-fallout-dialogue.json:26]:
```suggestion
{
  "intro": "猛烈な　煙と灰の嵐に　遭遇しました！ ",
  "title": "燃える熱気",
  "description": "渦巻く　灰と　残り火に　より、　視界は　ほぼゼロに　なりました。　どうやら　この状況を　引き起こしている　原因は　何かある　ようです…… しかし、これほどの規模の　現象を　起こしたのは　一体　何でしょうのか​​？",
  "query": "どうしますか？",
  "option": {
    "1": {
      "label": "原因を見つける",
      "tooltip": "(?) 原因を発見\n(-) キツいバトル",
      "selected": "火の嵐を　突き進むと、　交配の踊りを　している　2匹の　{{volcaronaName}} を　見つけます！$邪魔を快く思わず　襲いかかってきます！"
    },
    "2": {
      "label": "避難してみる",
      "tooltip": "(-) 気象状態に耐える",
      "selected": "避難場所を　探してみる　間に　気象状態が　重大な　被害を　もたらします！ $ 手持ちポケモンは　最大HPの　20％の　ダメージを受けます！",
      "target_burned": "その上、　{{burnedPokemon}} も　やけど状態　になりました！"
    },
    "3": {
      "label": "ほのおポケモンが助ける",
      "tooltip": "(+) 状態が治る\n(+) もくたんをもらう",
      "disabled_tooltip": "選ぶには　2匹の　ほのおタイプの　ポケモンが　必要",
      "selected": "{{option3PrimaryName}}と　{{option3SecondaryName}}が　交配の踊りを　している　2匹の　{{volcaronaName}}まで　導きます！$おかげさまで　落ち着かせる　ことができ\n{{volcaronaName}}が　問題なく　立ち去ります。"
    }
  },
  "found_charcoal": "気象状態が　治ってから　\n{{leadPokemon}}が　地面で　何かを　見つけます。$@s{item_fanfare}{{leadPokemon}}が　もくたんを　手に入れた！"
}
```

File: src/locales/ja/mystery-encounters/fight-or-flight-dialogue.json [hunk 2]
```diff
@@ -1 +1,25 @@
...
+    "2": {
+      "label": "Steal the Item",
+      "disabled_tooltip": "Your Pokémon need to know certain moves to choose this",
+      "tooltip": "(+) {{option2PrimaryName}} uses {{option2PrimaryMove}}",
+      "selected": ".@d{32}.@d{32}.@d{32}$Your {{option2PrimaryName}} helps you out and uses {{option2PrimaryMove}}!$You nabbed the item!"
+    },
+    "3": {
+      "label": "Leave",
+      "tooltip": "(-) No Rewards",
+      "selected": "You leave the strong Pokémon\nwith its prize and continue on."
```
Comment (Chapybara-jp) [inline src/locales/ja/mystery-encounters/fight-or-flight-dialogue.json:22]:
```suggestion
  "intro": "あの　ポケモンの　近くで　地面に何かが　きらきらと　輝いています！",
  "title": "闘争・逃走",
  "description": "アイテムを　守っている　強いポケモンが　いる　ようです。バトルは　直接的な　取り方ですが、　強そうです。　適切な　手持ちポケモンが　いれば、　アイテムを盗まれる　かもしれません。",
  "query": "どうしますか？",
  "option": {
    "1": {
      "label": "バトルする",
      "tooltip": "(-) 勝負がキツい\n(+) 新しいアイテム",
      "selected": "恐れないで　強いポケモンに　近寄ります。",
      "stat_boost": "{{enemyPokemon}}は　潜在的な　強さで　能力を　一つ　上げた！"
    },
    "2": {
      "label": "アイテムを盗む",
      "disabled_tooltip": "選ぶには　手持ちポケモンが　特定な　技を　覚えている必要　が　ある",
      "tooltip": "(+) {{option2PrimaryName}}が　 {{option2PrimaryMove}}を使う",
      "selected": ".@d{32}.@d{32}.@d{32}${{option2PrimaryName}}が　手を貸し　 {{option2PrimaryMove}}を　使います！$アイテムを　奪いました！"
    },
    "3": {
      "label": "立ち去る",
      "tooltip": "(-) ご褒美なし",
      "selected": "強いポケモンと　アイテムを　置いて　冒険を　進みます。"
```

File: src/locales/de/mystery-encounters/a-trainers-test-dialogue.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
   "option": {
     "1": {
       "label": "Die Herausforderung annehmen",
-      "tooltip": "(-) Schwerer Kampf\n(+) Erhalte ein @[TOOLTIP_TITLE]{Sehr seltenes Ei}"
+      "tooltip": "(-) Schwerer Kampf\n(+) Erhalte ein @[TOOLTIP_TITLE]{Sehr seltenes Ei}"//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/a-trainers-test-dialogue.json:34]:
```suggestion
      "tooltip": "(-) Sehr schwerer Kampf\n(+) Erhalte ein @[TOOLTIP_TITLE]{Sehr seltenes Ei}"
```

File: src/locales/de/mystery-encounters/bug-type-superfan-dialogue.json [hunk 4]
```diff
@@ -8,7 +8,7 @@
   "option": {
     "1": {
       "label": "Pokémon-Kampf",
-      "tooltip": "(-) Herausfordernder Kampf\n(+) Einem Pokémon eine Käfer-Attacke beibringen",
+      "tooltip": "(-) Herausfordernder Kampf\n(+) Einem Pokémon eine Käfer-Attacke beibringen",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/bug-type-superfan-dialogue.json:11]:
```suggestion
      "tooltip": "(-) Herausfordernder Kampf\n(+) Einem Pokémon eine Käfer-Attacke beibringen",
```

fine as is

File: src/locales/de/mystery-encounters/clowning-around-dialogue.json [hunk 5]
```diff
@@ -8,28 +8,28 @@
   "option": {
     "1": {
       "label": "Kampf beginnen",
-      "tooltip": "(-) Komischer Kampf\n(?) Beeinflusst Pokémon-Fähigkeiten",
+      "tooltip": "(-) Komischer Kampf\n(?) Beeinflusst Pokémon-Fähigkeiten",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/clowning-around-dialogue.json:11]:
```suggestion
      "tooltip": "(-) Komischer Kampf\n(?) Beeinflusst die Fähigkeit eines Pokémon",
```

File: src/locales/de/mystery-encounters/clowning-around-dialogue.json [hunk 6]
```diff
@@ -8,28 +8,28 @@
...
       "selected": "Deine erbärmlichen Pokémon sind bereit für eine erbärmliche Vorstellung!",
-      "apply_ability_dialogue": "Eine sensationelle Vorstellung! Dein Können passt zu einer sensationellen Fähigkeit als Beute!",
+      "apply_ability_dialogue": "Eine sensationelle Vorstellung! Dein Können passt zu einer sensationellen Fähigkeit als Beute!",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/clowning-around-dialogue.json:13]:
```suggestion
      "apply_ability_dialogue": "Eine sensationelle Vorstellung! Dein Können passt zu einer besonderen Fähigkeit als Beute!",
```

File: src/locales/de/mystery-encounters/clowning-around-dialogue.json [hunk 7]
```diff
@@ -8,28 +8,28 @@
...
       "apply_ability_message": "Der Clown bietet an, die Fähigkeit eines deiner Pokémon dauerhaft auf {{ability}} zu wechseln!",
       "ability_prompt": "Soll eines deiner Pokémon die Fähigkeit {{ability}} dauerhaft erlangen?",
       "ability_gained": "@s{level_up_fanfare}{{chosenPokemon}} hat die Fähigkeit {{ability}} erhalten!"
     },
     "2": {
       "label": "Nicht provozieren lassen",
-      "tooltip": "(-) Der Clown ist beleidigt\n(?) Beeinflusst Pokémon-Items",
+      "tooltip": "(?) Beeinflusst Pokémon-Items",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/clowning-around-dialogue.json:20]:
```suggestion
      "tooltip": "(?) Beeinflusst die Items eines Pokémons",
```

File: src/locales/de/mystery-encounters/clowning-around-dialogue.json [hunk 8]
```diff
@@ -8,28 +8,28 @@
...
       "selected":  "Du erbärmlicher Feigling, du verweigerst einen wunderbaren Kampf? Fühle meinen Zorn!",
       "selected_2": "Das {{blacephalonName}} des Clowns verwendet Trickbetrug! Alle Items deines {{switchPokemon}} wurden zufällig vertauscht!",
       "selected_3": "Meine perfekte List hat dich in die Irre geführt!"
     },
     "3": {
       "label": "Die Beleidigungen erwidern",
-      "tooltip": "(-) Den Clown verärgern\n(?) Beeinflusst Pokémon-Typen",
+      "tooltip": "(?) Beeinflusst Pokémon-Typen",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/clowning-around-dialogue.json:27]:
```suggestion
      "tooltip": "(?) Beeinflusst die Typen deiner  Pokémon",
```

File: src/locales/de/mystery-encounters/dancing-lessons-dialogue.json [hunk 9]
```diff
@@ -1,7 +1,7 @@
 {
   "intro": "Ein {{oricorioName}} tanzt traurig allein, ohne einen Partner.",
   "title": "Tanzstunden",
-  "description": "Das {{oricorioName}} scheint nicht aggressiv zu sein, im Gegenteil, es scheint traurig zu sein.\nVielleicht möchte es einfach nur mit jemandem tanzen...",
+  "description": "Das {{oricorioName}} scheint nicht aggressiv zu sein, im Gegenteil, es scheint traurig zu sein.\nVielleicht möchte es einfach nur mit jemandem tanzen...",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/dancing-lessons-dialogue.json:4]:
```suggestion
  "description": "Das {{oricorioName}} scheint nicht aggressiv zu sein, im Gegenteil, es scheint traurig zu sein.\nVielleicht möchte es einfach nur mit jemandem tanzen...",
```

Fine as is

File: src/locales/de/mystery-encounters/dancing-lessons-dialogue.json [hunk 10]
```diff
@@ -12,7 +12,7 @@
     },
     "2": {
       "label": "Lerne den Tanz",
-      "tooltip": "(+) Bringe einem Pokémon Wecktanz bei",
+      "tooltip": "(+) Bringe einem Pokémon Wecktanz bei",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/dancing-lessons-dialogue.json:15]:
```suggestion
      "tooltip": "(+) Bringe einem Pokémon Wecktanz bei",
```

fine as is

File: src/locales/de/mystery-encounters/delibirdy-dialogue.json [hunk 11]
```diff
@@ -1,7 +1,7 @@
 
 
 {
-  "intro": "Ein Schwarm {{delibirdName}} ist aufgetaucht!",
+  "intro": "Ein Schwarm {{delibirdName}} ist aufgetaucht!",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/delibirdy-dialogue.json:4]:
```suggestion
  "intro": "Ein Schwarm {{delibirdName}} ist aufgetaucht!",
```
fine as is

File: src/locales/de/mystery-encounters/delibirdy-dialogue.json [hunk 12]
```diff
@@ -25,5 +25,5 @@
       "selected": "Du wirfst {{chosenItem}} zu den {{delibirdName}}, die aufgeregt miteinander schnattern.$Sie drehen sich zu dir um und geben dir glücklich ein Geschenk!"
     }
   },
-  "outro": "Die {{delibirdName}} watscheln glücklich davon.$Was für ein seltsamer kleiner Austausch!"
-}
\ No newline at end of file
+  "outro": "Die {{delibirdName}} watscheln glücklich davon.$Was für ein seltsamer kleiner Austausch!"//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/delibirdy-dialogue.json:28]:
```suggestion
  "outro": "Die {{delibirdName}} watscheln glücklich davon.$Was für ein seltsamer kleiner Austausch!"
```

fine as is

File: src/locales/de/mystery-encounters/global-trade-system-dialogue.json [hunk 13]
```diff
@@ -11,13 +11,13 @@
     },
     "2": {
       "label": "Zaubertausch",
-      "tooltip": "(+) Seine eine deiner Pokémon an die GTS und erhalte ein zufälliges Pokémon im Austausch"
+      "tooltip": "(+) Seine eine deiner Pokémon an die GTS und erhalte ein zufälliges Pokémon im Austausch"//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/global-trade-system-dialogue.json:14]:
```suggestion
      "tooltip": "(+) Seine eine deiner Pokémon an die GTS und erhalte ein zufälliges besonderes Pokémon im Austausch"
```

File: src/locales/de/mystery-encounters/global-trade-system-dialogue.json [hunk 14]
```diff
@@ -11,13 +11,13 @@
...
     },
     "3": {
       "label": "Tausche ein Item",
       "trade_options_prompt": "Wähle ein Item aus, das du senden möchtest.",
       "invalid_selection": "Dieses Pokémon hat keine Items die getauscht werden können.",
-      "tooltip": "(+) Sende eines deiner Items an die GTS und erhalte ein zufälliges Item im Austausch"
+      "tooltip": "(+) Sende eines deiner Items an die GTS und erhalte ein zufälliges Item im Austausch"//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/global-trade-system-dialogue.json:20]:
```suggestion
      "tooltip": "(+) Sende eines deiner Items an die GTS und erhalte ein zufälliges verbessertes Item im Austausch"
```

File: src/locales/de/mystery-encounters/the-strong-stuff-dialogue.json [hunk 15]
```diff
@@ -1,7 +1,7 @@
 {
   "intro": "Es ist ein riesiger {{shuckleName}} und ein riesiger Vorrat an... Saft?",
   "title":  "Das gute Zeug",
-  "description": "Das {{shuckleName}} das deinen Weg blockiert, sieht unglaublich stark aus. In der Zwischenzeit strahlt der Saft daneben eine Art Kraft aus.\nDas {{shuckleName}} streckt seine Fühler in deine Richtung aus. Es scheint, als wolle es etwas tun...",
+  "description": "Das {{shuckleName}} das deinen Weg blockiert, sieht unglaublich stark aus. In der Zwischenzeit strahlt der Saft daneben eine Art Kraft aus.\nDas {{shuckleName}} streckt seine Fühler in deine Richtung aus. Es scheint, als wolle es etwas tun...",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/the-strong-stuff-dialogue.json:4]:
```suggestion
  "description": "Das {{shuckleName}} das deinen Weg blockiert, sieht unglaublich stark aus. In der Zwischenzeit strahlt der Saft daneben eine Art Kraft aus.\nDas {{shuckleName}} streckt seine Fühler in deine Richtung aus. Es scheint, als wolle es etwas tun...",
```

fine as is

File: src/locales/de/mystery-encounters/the-winstrate-challenge-dialogue.json [hunk 16]
```diff
@@ -3,12 +3,12 @@
   "speaker": "Die Sihgers",
   "intro_dialogue": "Wir sind die Sihgers!$Wie wäre es, wenn du gegen unsere Familie in einer Reihe von Pokémon-Kämpfen antrittst?",
   "title": "Die Sihgers-Herausforderung",
-  "description": "Die Sihgers sind eine Familie von 5 Trainern, und sie wollen kämpfen! Wenn du sie alle hintereinander besiegst, bekommst du einen grandiosen Preis. Aber kannst du die Hitze aushalten?",
+  "description": "Die Sihgers sind eine Familie von 5 Trainern, und sie wollen kämpfen! Wenn du sie alle hintereinander besiegst, bekommst du einen grandiosen Preis. Aber kannst du die Hitze aushalten?",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/the-winstrate-challenge-dialogue.json:6]:
```suggestion
  "description": "Die Sihgers sind eine Familie von @[TOOLTIP_TITLE]{5 Trainern}, und sie wollen kämpfen! Wenn du sie alle hintereinander besiegst, bekommst du einen grandiosen Preis. Aber kannst du die Hitze aushalten?",
```

File: src/locales/de/mystery-encounters/training-session-dialogue.json [hunk 17]
```diff
@@ -1,25 +1,25 @@
 {
   "intro": "Du stolperst über einige Trainingsutensilien und Vorräte.",
   "title": "Traningssitzung",
-  "description": "Diese Vorräte sehen so aus, als könnten sie verwendet werden, um ein Mitglied deines Teams zu trainieren! Es gibt ein paar Möglichkeiten, wie du dein Pokémon trainieren könntest, indem du gegen es mit dem Rest deines Teams kämpfst.",
+  "description": "Diese Vorräte sehen so aus, als könnten sie verwendet werden, um ein Mitglied deines Teams zu trainieren! Es gibt ein paar Möglichkeiten, wie du dein Pokémon trainieren könntest, indem du gegen es mit dem Rest deines Teams kämpfst.",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/training-session-dialogue.json:4]:
```suggestion
  "description": "Diese Vorräte sehen so aus, als könnten sie verwendet werden, um ein Mitglied deines Teams zu trainieren! Es gibt ein paar Möglichkeiten, wie du dein Pokémon trainieren könntest, indem du gegen es mit dem Rest deines Teams @[TOOLTIP_TITLE]{kämpfst und es besiegst}.",
```

File: src/locales/de/mystery-encounters/training-session-dialogue.json [hunk 18]
```diff
@@ -1,25 +1,25 @@
...
   "query": "Wie möchtest du trainieren?",
   "invalid_selection": "Pokémon muss genügend KP haben.",
   "option": {
     "1": {
       "label": "Leichtes Training",
-      "tooltip": "(-) Leichter Kampf\n(+) Verbessere 2 zufällige IS-Werte des Pokémon",
+      "tooltip": "(-) Leichter Kampf\n(+) Verbessere 2 zufällige IS-Werte des Pokémon",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/training-session-dialogue.json:10]:
```suggestion
      "tooltip": "(-) Leichter Kampf\n(+) Verbessere 2 zufällige IS-Werte des Pokémon",
```

fine as is

File: src/locales/de/mystery-encounters/training-session-dialogue.json [hunk 19]
```diff
@@ -1,25 +1,25 @@
...
       "tooltip": "(-) Moderater Kampf\n(+) Ändere das Wesen des Pokémon",
-      "select_prompt": "Wähle ein neues Wesen aus, um dein Pokémon zu trainieren.",
+      "select_prompt": "Wähle ein neues Wesen aus, um dein Pokémon zu trainieren.",//TODO
       "finished": "{{selectedPokemon}} kehrt zurück, fühlt sich erschöpft aber zufrieden!$Es hat nun ein neues Wesen: {{nature}}!"
     },
     "3": {
       "label": "Schweres Training",
       "tooltip": "(-) Harter Kampf\n(+) Ändere die Fähigkeit des Pokémon",
-      "select_prompt": "Wähle eine neue Fähigkeit aus, um dein Pokémon zu trainieren.",
+      "select_prompt": "Wähle eine neue Fähigkeit aus, um dein Pokémon zu trainieren.",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/training-session-dialogue.json:22]:
```suggestion
      "select_prompt": "Wähle eine neue Fähigkeit aus, um dein Pokémon zu trainieren.",
```

fine as is

File: src/locales/de/mystery-encounters/training-session-dialogue.json [hunk 20]
```diff
@@ -1,25 +1,25 @@
...
       "label": "Leichtes Training",
-      "tooltip": "(-) Leichter Kampf\n(+) Verbessere 2 zufällige IS-Werte des Pokémon",
+      "tooltip": "(-) Leichter Kampf\n(+) Verbessere 2 zufällige IS-Werte des Pokémon",//TODO
       "finished": "{{selectedPokemon}} kommt zurück, fühlt sich erschöpft aber zufrieden!$Seine {{stat1}} und {{stat2}} IS-Werte wurden verbessert!"
     },
     "2": {
       "label": "Moderates Training",
       "tooltip": "(-) Moderater Kampf\n(+) Ändere das Wesen des Pokémon",
-      "select_prompt": "Wähle ein neues Wesen aus, um dein Pokémon zu trainieren.",
+      "select_prompt": "Wähle ein neues Wesen aus, um dein Pokémon zu trainieren.",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/training-session-dialogue.json:16]:
```suggestion
      "select_prompt": "Wähle ein neues Wesen aus, um dein Pokémon zu trainieren.",
```
fine as is

File: src/locales/de/mystery-encounters/trash-to-treasure-dialogue.json [hunk 21]
```diff
@@ -6,7 +6,7 @@
   "option": {
     "1": {
       "label": "Nach Wertsachen suchen",
-      "tooltip": "(-) Heilitems kosten ab jetzt das Dreifache\n(+) Erhalte tolle Items",
+      "tooltip": "(-) Heilitems kosten ab jetzt das Dreifache\n(+) Erhalte tolle Items",//TODO
```
Comment (CodeTappert) [inline src/locales/de/mystery-encounters/trash-to-treasure-dialogue.json:9]:
```suggestion
      "tooltip": "(-) Heilitems kosten ab jetzt {{costMultiplier}} mal so viel\n(+) Erhalte tolle Items",//TODO
```

**Review by Chapybara-jp — APPROVED** [review]

**Comment by Chapybara-jp** [comment]
Japanese translation is now complete!

**Review by SangaraSorama — APPROVED** [review]

**Pull request merged**